finality-aleph/src/party/mod.rs

```diff
@@ -263,9 +293,13 @@ where
     metrics: Option<Metrics<<B::Header as Header>::Hash>>,
     authority_justification_tx: mpsc::UnboundedSender<JustificationNotification<B>>,
     unit_creation_delay: UnitCreationDelay,
+    authorities: Vec<aleph_primitives::Public>,
 }
 
 const SESSION_STATUS_CHECK_PERIOD_MS: u64 = 1000;
+const SESSION_RESTART_COOLDOWN_MS: u64 = 500;
+
+type SplitData<B> = Split<AlephNetworkData<B>, RmcNetworkData<B>>;
```

Comment:
Move this slightly higher, so that it at least appears before the first usage. Perhaps to the top of the file after all. ">_>

Reply:
Done